qiskit/utils/arithmetic.py

```diff
@@ -131,3 +131,24 @@ def transpositions(permutation: List[int]) -> List[Tuple[int, int]]:
             for y in x[len_x - 1:0:-1]:
                 res.append((first, y))
     return res
+
+
+def _triu_to_dense(triu: np.ndarray) -> np.ndarray:
+    """
+    Converts upper triangular part of matrix to dense matrix.
```

Comment:
Let's match the overall Qiskit documentation style 🙂 
```suggestion
    """Converts upper triangular part of matrix to dense matrix.
```